dev-packages/e2e-tests/test-applications/solidstart/app.config.ts

```diff
@@ -1,3 +1,8 @@
+import { sentrySolidStartVite } from '@sentry/solidstart';
 import { defineConfig } from '@solidjs/start/config';
 
-export default defineConfig({});
+export default defineConfig({
+  vite: {
+    plugins: [sentrySolidStartVite({})],
```

Comment:
l/just personal opinion: It feels a bit awkward that users need to pass an options object but can leave it empty. Should we make the options object optional? I might be missing something but technically, I could pass all required options for source maps upload as env variables, right? (project slug and auth token)

Reply:
Right, good point! I'll make this optional.

Reply:
@Lms24 updated